pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentStickyKeyDispatcherMultipleConsumersClassic.java

```diff
@@ -173,8 +191,11 @@ private void sortRecentlyJoinedConsumersIfNeeded() {
                 posPre = posAfter;
             }
         }
-
         if (sortNeeded) {
+            log.error("[{}] [{}] The items in recentlyJoinedConsumers are out-of-order. {}",
+                    topic.getName(), name, recentlyJoinedConsumers.entrySet().stream().map(entry ->
+                            String.format("%s-%s:%s", entry.getKey().consumerName(), entry.getValue().getLedgerId(),
+                                    entry.getValue().getEntryId())).collect(Collectors.toList()));
```

Comment:
I think we should use warning or info log here since it is expected.

Reply:
After this PR, this is unexpected behavior.

Reply:
> After this PR, this is unexpected behavior.

+1